lib/infra_conversion_throttler.rb

```diff
@@ -34,7 +34,8 @@ def self.start_conversions
 
         eligible_host = eligible_hosts.first
         _log.debug("-- Associating  #{eligible_host.name} to the task for '#{vm_name}'.")
-        job.migration_task.update_attributes!(:conversion_host => eligible_hosts.first)
+        job.migration_task.update!(:conversion_host => eligible_host)
```

Comment:
Since we're not checking the return value, I'm curious what the difference is here between `update` and `update_attributes`. Not that I have a problem with it, was just curious what prompted the change.

Reply:
`update_attributes` is deprecated in favor of `update`.  Probably changed due to a rubocop comment.

Reply:
@bdunne Ah, thanks.